hw/ip/otp_ctrl/data/BUILD

```diff
@@ -155,27 +168,43 @@ otp_json(
             items = {
                 # Enable bootstrap. See `hardened_bool_t` in
                 # sw/device/lib/base/hardened.h.
-                "OWNER_SW_CFG_ROM_BOOTSTRAP_DIS": "0x1d4",
+                "OWNER_SW_CFG_ROM_BOOTSTRAP_DIS": otp_hex(CONST.FALSE),
                 # Set to 0x739 to use the ROM_EXT hash measurement as the key
                 # manager attestation binding value.
                 "OWNER_SW_CFG_ROM_KEYMGR_ROM_EXT_MEAS_EN": "0x0",
                 # Report errors without any redaction.
-                "OWNER_SW_CFG_ROM_ERROR_REPORTING": "0xe2290aa5",
+                "OWNER_SW_CFG_ROM_ERROR_REPORTING": otp_hex(CONST.SHUTDOWN.REDACT.NONE),
                 # Set the enables to kAlertEnableNone.
                 # See `alert_enable_t` in
                 # sw/device/silicon_creator/lib/drivers/alert.h
-                "OWNER_SW_CFG_ROM_ALERT_CLASS_EN": "0xa9a9a9a9",
-                # Set the esclation policies to kAlertEscalateNone.
+                "OWNER_SW_CFG_ROM_ALERT_CLASS_EN": otp_bytestring([
+                    CONST.ALERT.NONE, # Class A
+                    CONST.ALERT.NONE, # Class B
+                    CONST.ALERT.NONE, # Class C
+                    CONST.ALERT.NONE, # Class D
+                ]),
+                # Set the escalation policies to kAlertEscalateNone.
                 # See `alert_escalate_t` in
                 # sw/device/silicon_creator/lib/drivers/alert.h
-                "OWNER_SW_CFG_ROM_ALERT_ESCALATION": "0xd1d1d1d1",
-                # Set the classifiactions to kAlertClassX.
+                "OWNER_SW_CFG_ROM_ALERT_ESCALATION": otp_bytestring([
+                    CONST.ALERT.ESC_NONE, # Class A
+                    CONST.ALERT.ESC_NONE, # Class B
+                    CONST.ALERT.ESC_NONE, # Class C
+                    CONST.ALERT.ESC_NONE, # Class D
+                ]),
+                # Set the classifications to kAlertClassX.
                 # See `alert_class_t` in
                 # sw/device/silicon_creator/lib/drivers/alert.h
-                "OWNER_SW_CFG_ROM_ALERT_CLASSIFICATION": ["0x94949494"] * 80,
-                # Set the classifiactions to kAlertClassX. See `alert_class_t` in
+                "OWNER_SW_CFG_ROM_ALERT_CLASSIFICATION": otp_alert_classification(
+                    default = otp_bytestring([CONST.ALERT.CLASS_X] * 4),
+                    num_alerts = 80,
+                ),
+                # Set the classifications to kAlertClassX. See `alert_class_t` in
```

Comment:
I think ultimately, the alert configuration is going to have to be product (ie: earlgrey) specific and will need a way to name all of the alerts (see `top_earlgrey_alert_id` in `top_earlgrey.h`).  I think this going to have to look something like:
```
"OWNER_SW_CFG_ROM_ALERT_CLASSIFICATION": otp_earlgrey_alert_classification(
  uart0_fatal_fault = alert(prod="A", prod_end="A", dev="B", rma="X"),
  uart1_fatal_fault = alert(prod="A", prod_end="A", dev="B", rma="X"),
  ... repeats for all alerts ...
),
"OWNER_SW_CFG_ROM_LOCAL_ALERT_CLASSIFICATION": otp_earlgrey_local_alert_classification(
  ping_fail = alert(prod="A", prod_end="A", dev="B", rma="X"),
  escalation_fail = alert(prod="A", prod_end="A", dev="B", rma="X"),
  ... repeats for all local alerts ...
),
```